db/migrations/20190123095220_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.raw('UPDATE movies SET name = title');
+};
+
+exports.down = (Knex, Promise) => {
+  return Promise.resolve();
+};
```

Comment:
keep an eye out for these! looks like your editor keeps removing trailing new lines. all files should end in a newline.